pkg/cmd/build/new.go

```diff
@@ -0,0 +1,88 @@
+package build
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/buildkite/cli/v3/internal/io"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/buildkite/go-buildkite/v3/buildkite"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdBuildNew(f *factory.Factory) *cobra.Command {
+	var message string
+	var commit string
+	var branch string
+	var web bool
+
+	cmd := cobra.Command{
+		DisableFlagsInUseLine: true,
+		Use:                   "new <pipeline> [flags]",
+		Short:                 "Creates a new pipeline build",
+		Args:                  cobra.ArbitraryArgs,
```

Comment:
I think this should be:
```suggestion
		Args:                  cobra.ExactArgs(1),
```

Reply:
I'll make this change together with the suggestions below.